fanuc_lrmate200id_moveit_config/package.xml

```diff
@@ -29,6 +29,7 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
+  <exec_depend>fanuc_lrmate200id_gazebo</exec_depend>
```

Comment:
There was some debate over whether the `mpe_gazebo.launch` file should actually not be in a different package, like the `abb_irb120_gazebo` package itself. Problem is that that package has no dependency on MoveIt, and it doesn't need it either, as it's perfectly possible to use Gazebo without MoveIt.

An alternative could be to create a *new* package, `abb_irb120_gazebo_moveit_config` which would contain *just* the `mpe_gazebo.launch` file, and `exec_depend` both on `abb_irb120_moveit_config` and `abb_irb120_gazebo`.

It would cleanly separate the dependencies, but a drawback is that it would add `1 + N` packages for each `N` variants we want to add Gazebo support for.

Any ideas?

@shaun-edwards?